ui/window.go

```diff
@@ -189,6 +190,20 @@ func (win *Window) HandleEvents() {
 		win.Quit <- struct{}{}
 	}
 
+	// Create window chan event and listen events from window event
+	events := make(chan event.Event)
+	acks := make(chan struct{})
```

Comment:
```suggestion
	acks := make(chan struct{},2)
```

Making `acks` channel buffered and have capacity of 2, prevents the channel blocking that happens when `DestroyEvent` is detected. For all the other events you write once to `acks` channel before reading but when `DestroyEvent` is detected you attempt to write to the channel twice. The second write will block the app shutdown if a buffered channel is not used.